src/main/java/com/bunchiestudios/cahserver/requests/CreateGameRequest.java

```diff
@@ -3,21 +3,83 @@
 import com.bunchiestudios.cahserver.RequestIdentifier;
 import com.bunchiestudios.cahserver.ServerRequest;
 import com.bunchiestudios.cahserver.database.DataManager;
+import com.bunchiestudios.cahserver.database.InvalidQueryException;
+import com.bunchiestudios.cahserver.datamodel.Card;
+import com.bunchiestudios.cahserver.datamodel.Game;
+import com.bunchiestudios.cahserver.datamodel.GameCard;
+import com.bunchiestudios.cahserver.datamodel.Player;
+import com.github.fge.jsonschema.core.exceptions.ProcessingException;
 import com.twitter.finagle.http.Method;
 import org.json.JSONObject;
 
+import java.util.ArrayList;
+import java.util.List;
+
 /**
- * Created by rdelfin on 8/27/16.
+ * This is the request made by the user when they want to create a new game. Provided a game name, an optional password,
+ * and some authentication, the appropriate data structures should be set up.
  */
 public class CreateGameRequest extends ServerRequest {
 
-    public CreateGameRequest(DataManager mgr) {
-        super(mgr);
+    public CreateGameRequest(DataManager mgr) throws ProcessingException {
+        super(mgr, "resource:json/schema/request/CreateGame.json", "resource:json/schema/response/CreateGame.json");
```

Comment:
wouldn't it be easier if you move all of this hard coded crap to a constants file?